### PR TITLE
Fix conversion warning for module base address

### LINKED ISSUE #5: ULONGLONG to uintptr_t conversion causes DEBUG warning
The following warning is produced only when compiling in debug mode. The warning does not appear to exist in ReleaseMinSize builds.

Warning	C4244	'=': conversion from 'ULONGLONG' to 'uintptr_t', possible loss of data	NetHelper	\nethelper\src\patcher.cpp	570

Produced by the line ->ImageBase below.

``` c++
uintptr_t preferredAddress;

 . . .

    else if (optionalHeader->Magic == IMAGE_NT_OPTIONAL_HDR64_MAGIC) {
      preferredAddress = reinterpret_cast<IMAGE_OPTIONAL_HEADER64*>(optionalHeader)
                         ->ImageBase;
    }

. . .

modulePrefAddr[module] = preferredAddress;
```

### PR DESCRIPTION
**Pull request by DanRStevens**
Uses conditional compilation to prevent dead code.

Please check the indentation of the sanity checking `if` statement. It's multi-line, and I aligned it, which may be bad practice.

Closes #5.

### CODE REVIEW

File: src/Patcher.cpp [hunk 1]
```diff
@@ -562,17 +562,17 @@ void* FixPtr(const void *pointer, HMODULE module) {
...
+    if ((optionalHeader->Magic != IMAGE_NT_OPTIONAL_HDR32_MAGIC) && 
+       ((optionalHeader->Magic != IMAGE_NT_OPTIONAL_HDR64_MAGIC)) {
       // Not a valid PE image
       return nullptr;
-    }
+		}
+
+    #ifdef _WIN64
+      preferredAddress = reinterpret_cast<IMAGE_OPTIONAL_HEADER64*>(optionalHeader)->ImageBase;
+    #elif
```
Comment (Brett208) [inline src/Patcher.cpp]: I believe this should be #else. On MSVC, a compiler error of E0127, expected a statement is given.

```suggestion
    #else
```

**Comment by Brett208** [comment]
I made a commit that removed an extra parenthesis. I also tried to clean up the whitespace a little to match the current structure.

I think the #elif command is supposed to be #else. I tried the new suggest change feature of GitHub. It is supposed to give me credit for the commit somehow? Maybe something worth toying with for fun. Happy if you just commit on your own too though.

I already tested by replacing #elif by #else and everything compiles without warning. Shows external IP address inside Outpost 2. So this fixes the warning. I'd like to see it committed after the suggested change is made.

-Brett

File: src/Patcher.cpp (hunk 1 above)
Comment (DanRStevens) [reply src/Patcher.cpp]: Ahh yes, thanks for catching that. I was translating from the original `if` - `else if` - `else` structure, and didn't update it.

Neat little suggest mechanism.

**Comment by DanRStevens** [comment]
Thank you for the testing and updates. Looks like I'd left a bit of a mess with the parenthesis and indentation.

**Pull request merged**